src/app/MessageDef/CommandStatusIB.h

```diff
@@ -86,6 +98,15 @@ class Builder : public StructBuilder
      */
     StatusIB::Builder & CreateErrorStatus();
 
+    /**
+     *  @brief Inject Command Ref into the TLV stream.
+     *
+     *  @param [in] aRef refer to the CommandRef to set in CommandStatusIB.
+     *
+     *  @return #CHIP_NO_ERROR on success
+     */
+    CHIP_ERROR Ref(const uint16_t aRef);
```

Comment:
not need for const?  just pass by value?

Reply:
I am just following convention of other builders https://github.com/project-chip/connectedhomeip/blob/master/src/app/MessageDef/ClusterPathIB.h#L101. IIRC `const` here would be compile time checked and doesn't add any overhead